e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/change-your-answers/change-your-answers-change-policy-type-single-to-multiple.spec.js

```diff
@@ -0,0 +1,141 @@
+import { submitButton } from '../../../../pages/shared';
+import { typeOfPolicyPage, checkYourAnswersPage } from '../../../../pages/insurance/policy-and-export';
+import partials from '../../../../partials';
+import { FIELD_IDS, FIELD_VALUES, ROUTES } from '../../../../../../constants';
+import { DEFAULT, LINKS } from '../../../../../../content-strings';
+import { POLICY_AND_EXPORT_FIELDS as FIELDS } from '../../../../../../content-strings/fields/insurance/policy-and-exports';
+import { INSURANCE_ROOT } from '../../../../../../constants/routes/insurance';
+import getReferenceNumber from '../../../../helpers/get-reference-number';
+
+const {
+  POLICY_AND_EXPORTS: {
+    CHECK_YOUR_ANSWERS,
+    TYPE_OF_POLICY_CHANGE,
+    MULTIPLE_CONTRACT_POLICY_CHANGE,
+  },
+} = ROUTES.INSURANCE;
+
+const {
+  INSURANCE: {
+    POLICY_AND_EXPORTS: {
+      TYPE_OF_POLICY: { POLICY_TYPE },
+      CONTRACT_POLICY: {
+        MULTIPLE: {
+          TOTAL_MONTHS_OF_COVER,
+          TOTAL_SALES_TO_BUYER,
+          MAXIMUM_BUYER_WILL_OWE,
+        },
+      },
+    },
+  },
+} = FIELD_IDS;
+
+const { CONTRACT_POLICY: { MULTIPLE: MULTIPLE_FIELD_STRINGS } } = FIELDS;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.prepareApplication.tasks.policyTypeAndExports;
+
+const { summaryList } = checkYourAnswersPage;
+
+context('Insurance - Policy and exports - Change your answers - Policy type - single to multiple', () => {
+  let referenceNumber;
+
+  before(() => {
+    cy.navigateToUrl(ROUTES.INSURANCE.START);
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    task.link().click();
+
+    cy.completeAndSubmitPolicyTypeForm(FIELD_VALUES.POLICY_TYPE.SINGLE);
+    cy.completeAndSubmitSingleContractPolicyForm();
+    cy.completeAndSubmitAboutGoodsOrServicesForm();
+
+    getReferenceNumber().then((id) => {
+      referenceNumber = id;
+
+      const expected = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${CHECK_YOUR_ANSWERS}`;
+      cy.url().should('eq', expected);
+    });
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('connect.sid');
+  });
+
+  const fieldId = POLICY_TYPE;
+
+  describe('when clicking the `change` link', () => {
+    before(() => {
+      summaryList[fieldId].changeLink().click();
+    });
+
+    it(`should redirect to ${TYPE_OF_POLICY_CHANGE}`, () => {
+      const expected = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${TYPE_OF_POLICY_CHANGE}#heading`;
```

Comment:
Yes 😊 In all "change" links  in the "check your answers" page, we add a hash tag to each URL - usually `#label` so that when changing the answer, the user has full context of what the field is (rather than going direct to the input and auto focusing, this causes accessibility issues).

For "type of policy", it's `#heading` because on this page, there's only 1 form field (2 radio buttons) and the legend/label for these radios, is the page heading